### PR TITLE
Created a Function and a Unit Test

### PR DESCRIPTION
**Pull request by bobbyesh**
## Purpose
To add a sample function and test to be used in demonstrating our workflow.

## Changes

- Added a hypotenuse function
- Added a unit test
- Added some documentation
- Added a Cargo.toml

## Implementation Decisions and Issues

I opted for a math function because testing for stdout in Rust turned out to not be the least time consuming thing in the world. The options are using either a  third-party package to stub out stdout or to use the `Command` module in stdlib to have the program call itself and test what comes out of stdout. I tried to pull this off, but unsurprisingly got an infinite loop of the program calling itself and was not able to see how my code differed from the example code I had found at a glance.

I did not include documentation tests (tests in the comments) because after some time, I was still getting compilation errors.

### TL;DR

Used no print function and no doctest because time.

## Relevant Information For CI

The command for running the tests is:
```
$ cargo test
```

Output:
```
   Compiling demo v0.1.0 (file:///~/projects/integrationsDemo)
    Finished debug [unoptimized + debuginfo] target(s) in 2.98 secs
     Running target/debug/demo-6f67ed345d78cd91

running 1 test
test tests::hypotenuse_test ... ok

test result: ok. 1 passed; 0 failed; 0 ignored; 0 measured
```

...which can be called in the project root directory.

## Feedback

-  Should we use something other than hypotenuse?
-  Should it be a real "Hello World" function?
-  Code style?  Opt for imperative style functions instead of functional programming style functions?
-  Did we want documentation tests?

### CODE REVIEW

**Comment by kneitinger** [comment]
This is a fantastic PR Bobby, well done!

Re your Feedback section:
1) we should write a few more tests and familiarize ourselves!
3) No pref. here :)
4) We should probably look into that before getting on to our legit repo

Merging it in, so we can begin the TravisCI stuff.
